packages/kbn-apm-config-loader/src/config.ts

```diff
@@ -0,0 +1,116 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { join } from 'path';
+import { merge, get } from 'lodash';
+import { execSync } from 'child_process';
+// deep import to avoid loading the whole package
+import { getDataPath } from '@kbn/utils/target/path';
+import { readFileSync } from 'fs';
+
+const defaultConfig = {
+  active: false,
+  serverUrl: 'https://f1542b814f674090afd914960583265f.apm.us-central1.gcp.cloud.es.io:443',
+  // The secretToken below is intended to be hardcoded in this file even though
+  // it makes it public. This is not a security/privacy issue. Normally we'd
+  // instead disable the need for a secretToken in the APM Server config where
+  // the data is transmitted to, but due to how it's being hosted, it's easier,
+  // for now, to simply leave it in.
+  secretToken: 'R0Gjg46pE9K9wGestd',
+  globalLabels: {},
+  breakdownMetrics: true,
+  centralConfig: false,
+  logUncaughtExceptions: true,
+};
+
+export class ApmConfiguration {
+  private baseConfig?: any;
+  private kibanaVersion: string;
+
+  constructor(
+    private readonly rootDir: string,
```

Comment:
You can get this from `ROOT_DIR` in `@kbn/utils`

Reply:
@tylersmalley Just thought about something after our zoom meeting: If you were planning to move the `src/apm` script to `@kbn/utils` and import `@kbn/apm-config-loader` from here, We are going to cause a cyclic dependency, as `@kbn/apm-config-loader` actually also got a dependency to `@kbn/utils`.

Maybe the `src/apm` script itself should be moved to `@kbn/apm-config-loader` instead to avoid this? In that case, I could rename it to something like `@kbn/apm-agent` for example. Would not change much for future consumers of the agent or agent config. WDYT?